### PR TITLE
Community Links.de.md

### PR DESCRIPTION
**Pull request by Asgaror**

### CODE REVIEW

File: content/statement/Skycoin Community Links.de.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+* [Skycoin Blog](https://blog.skycoin.net)
+* [Skycoin Twitter](https://twitter.com/skycoinproject)
+* [Skycoin Reddit](https://reddit.com/r/skycoinproject)
+
+Software
+--------
+
+Der gesamte Code ist auf Github veröffentlicht. Mitarbeiter und Tester sind angehalten mitzuwirken.
+
```
Comment (AndersNiggelson) [inline content/statement/Skycoin Community Links.de.md]: -ist +wird

File: content/statement/Skycoin Community Links.de.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+* [Skycoin Slack](https://skycoin.herokuapp.com)
+
+Neuigkeiten und Updates 
+------------
+
+* [Skycoin Blog](https://blog.skycoin.net)
+* [Skycoin Twitter](https://twitter.com/skycoinproject)
+* [Skycoin Reddit](https://reddit.com/r/skycoinproject)
+
```
Comment (AndersNiggelson) [inline content/statement/Skycoin Community Links.de.md]: * [Skycoin Facebook](https://www.facebook.com/skycoinproject)


Medien
-----

* [Skycoin Youtube](https://www.youtube.com/channel/UCzLASufel2No4vSt4rudHSQ)
* [Skycoin Instagram](https://www.instagram.com/skycoinproject/)

File: content/statement/Skycoin Community Links.de.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+* [Skycoin Telegram Channel](https://t.me/Skycoin)
+* [Skycoin BBS Channel](https://t.me/skycoinbbs)
+* [Skycoin News Channel](https://t.me/skycoinnews)
+* [Skycoin Bounty Program Channel](https://t.me/skycoinbounty)
+
+Es gibt außerdem einen Slack-Channel
+
+* [Skycoin Slack](https://skycoin.herokuapp.com)
+
```
Comment (AndersNiggelson) [inline content/statement/Skycoin Community Links.de.md]: +* [Skycoin Discord](https://discord.gg/MTepVHE)

Mailingliste
------------

* [Skycoin Mailing List](http://eepurl.com/c4DyAv)

File: content/statement/Skycoin Community Links.de.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+categories = [
+    "Statement",
+]
++++
+
+Gespräch mit dem Skycoin Team
+--------------------------
+
+Das Skycoin Team ist das aktivste auf Telegram.
+
```
Comment (AndersNiggelson) [inline content/statement/Skycoin Community Links.de.md]: "Spreche mit dem Skycoin Team" oder "Kontaktiere das Skycoin Team"

"Das Skycoin Team ist am aktivsten auf Telegram."

**Comment by Asgaror** [comment]
Thanks for reviewing, i implemented the suggested changes + double checked the updates with the source file.

**Comment by gz-c** [comment]
1 sky

**Pull request merged**